staging/src/k8s.io/apimachinery/pkg/util/net/interface.go

```diff
@@ -254,13 +254,13 @@ func getIPFromInterface(intfName string, forFamily AddressFamily, nw networkInte
 		if err != nil {
 			return nil, err
 		}
-		klog.V(4).Infof("Interface %q has %d addresses :%v.", intfName, len(addrs), addrs)
+		klog.V(4).InfoS("Interface's addresses", "interfaceName", intfName, "addressesNum", len(addrs), "addresses", addrs)
 		matchingIP, err := getMatchingGlobalIP(addrs, forFamily)
 		if err != nil {
 			return nil, err
 		}
 		if matchingIP != nil {
-			klog.V(4).Infof("Found valid IPv%d address %v for interface %q.", int(forFamily), matchingIP, intfName)
+			klog.V(4).InfoS("Found valid address for interface", "ipVersion", int(forFamily), "address", matchingIP, "interfaceName", intfName)
```

Comment:
The InfoS output is less readable. How can a reader know the relationship between the `ipVersion`, `address`, and `interfaceName`?